Khan/perseus
# [Numeric Input] - Update styling to match modern inputs

## Summary:
The styling of the numeric input widget is quite constrained in regards to its size, and it is difficult to know when it has focus. Increasing its height will make it more physically accessible (better touch target). Adding more internal padding and increasing the border width will help with visual accessibility regarding a user's recognition of it having focus. Also, matching the styling that is used for the Expression widget (a close relative of this widget) lessens the cognitive load on users who see these two inputs as being similar/same (they don't know that they are two different widgets).

Issue: LEMS-2426

## Test plan:
1. Launch Storybook (yarn start)
1. Navigate to Perseus => Widgets => NumericInput => [Question 1](http://localhost:6006/?path=/story/perseus-widgets-numericinput--question-1)
   * Note that the input field is larger
1. Click or tab into the field
   * Note that the border is thicker and that the cursor is more visible (not next to the border as it was before)

## Affected UI:
### Standard presentation
#### Before:
![Before - Standard](https://github.com/user-attachments/assets/37644eb2-614d-4c8e-9d45-8222b47d03a4)

#### After:
![After - Standard](https://github.com/user-attachments/assets/bf9730af-55fa-4015-8d7e-c66fe20ace22)

### In-Focus
#### Before:
![Before - Focused](https://github.com/user-attachments/assets/9d9e786b-d152-4d92-bb11-8d7eff025743)

#### After:
![After - Focused](https://github.com/user-attachments/assets/46028bd9-892a-46a4-a9c0-f2e4227bfd40)
